drivers/include/net/netdev.h

```diff
@@ -283,7 +284,7 @@ typedef void (*netdev_event_cb_t)(netdev_t *dev, netdev_event_t event);
 struct netdev {
     const struct netdev_driver *driver;     /**< ptr to that driver's interface. */
     netdev_event_cb_t event_callback;       /**< callback for device events */
-    void* context;                          /**< ptr to network stack context */
+    void *context;                          /**< ptr to network stack context */
```

Comment:
I agree with the style changed. But this is not related to the PR and should be done separately.

Reply:
Alright, so I will not touch that.

Reply:
You could rebase, it is now in master ;-)